torch/nn/utils/clip_grad.py

```diff
@@ -25,9 +25,12 @@ def clip_grad_norm_(parameters, max_norm, norm_type=2):
     max_norm = float(max_norm)
     norm_type = float(norm_type)
     if norm_type == inf:
```

Comment:
nit: Do we really want to keep that special case? We already have it in the implementation of norm.

Reply:
Good point, I'll remove it.